### PR TITLE
DRIVERS-2265: Tests for aggregate allowDiskUse option

### PR DESCRIPTION
**Pull request by jmikola**
https://jira.mongodb.org/browse/DRIVERS-2265

PHPLIB POC: https://github.com/mongodb/mongo-php-library/pull/913

### CODE REVIEW

File: source/crud/tests/unified/aggregate-allowdiskuse.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  "initialData": [
+    {
+      "collectionName": "coll0",
+      "databaseName": "crud-tests",
+      "documents": []
+    }
+  ],
+  "tests": [
+    {
+      "description": "Aggregate does not send allowDiskuse when value is not specified",
```
Comment (jyemin) [inline source/crud/tests/unified/aggregate-allowdiskuse.json:37]: Nit:  allowDiskuse -> allowDiskUse

in all three tests.

**Review by jyemin — APPROVED** [review]
Happened to see this and was curious if Java driver already passes the new tests: it does.

One nit, but LGTM either way.

**Review by benjirewis — APPROVED** [review]
Passes in the Go driver; LGTM mod @jyemin's typo callout.

File: source/crud/tests/unified/aggregate-allowdiskuse.json (hunk 1 above)
Comment (jmikola) [reply source/crud/tests/unified/aggregate-allowdiskuse.json]: Fixed, as well as a previous typo in the `find` test file.

**Pull request merged**